test/relations.test.js

```diff
@@ -3790,7 +3790,7 @@ describe('relations', function() {
           return Person.findById(personId);
         })
         .then(function(person) {
-          person.passportItem().toObject().should.eql({name: 'Jim'});
+          person.passportItem().toObject().should.eql({name: 'Mitsos'});
```

Comment:
I see, thank you for looking up the original patch. My conclusion is that the name should indeed be "Jim".

 Honestly, I don't see any obvious reason explaining why you are seeing the original value "Mitsos" now.